R/standalone-obj-type.R

```diff
@@ -311,10 +314,15 @@ stop_input_type <- function(x,
   if (length(what)) {
     what <- oxford_comma(what)
   }
+  if (inherits(arg, "AsIs")) {
+    format_arg <- identity
```

Comment:
I'm not sure if this should be `unclass()`. `abort()` does ignore unmatching `arg` but maybe the `AsIs` class could be detected to ignore it unconditionally.